ICAFIX/ApplyAutoRecleanPipeline.sh

```diff
@@ -27,10 +27,9 @@ opts_AddOptional '--mrfix-concat-name' 'MRFixConcatName' 'rfMRI_REST' "if multi-
 opts_AddMandatory '--fix-high-pass' 'HighPass' 'integer' 'the high pass value that was used when running FIX' '--melodic-high-pass'
 opts_AddMandatory '--fmri-resolution' 'fMRIResolution' 'string' "resolution of data, like '2' or '1.60'"
 opts_AddMandatory '--subject-expected-timepoints' 'subjectExpectedTimepoints' 'string' "output spectra size for sICA individual projection, RunsXNumTimePoints, like '4800'"
-#TSC: doesn't default to MSMAll because we don't have that default string in the MSMAll pipeline
-opts_AddMandatory '--surf-reg-name' 'RegName' 'MSMAll' "the registration string corresponding to the input files"
 opts_AddConfigMandatory '--low-res' 'LowResMesh' 'LowResMesh' 'meshnum' "mesh resolution, like '32' for 32k_fs_LR"
-opts_AddOptional '--python-singularity' 'PythonSingularity' 'string' "the file path of the singularity" "$HCPPIPEDIR/ArealClassifier/hcp_python_singularity.simg"
+opts_AddOptional '--python-singularity' 'PythonSingularity' 'string' "the file path of the singularity, specify 'NONE' or empty string to use native environment instead" ""
```

Comment:
I'd rather not support "NONE" as a synonym for empty string in any new pipeline going forward.